### PR TITLE
[jsk_fetch_startup] fix odom topic of move_base and teleop

### PR DESCRIPTION
**Pull request by knorth55**
this PR depends on https://github.com/fetchrobotics/fetch_ros/pull/137
update odom topic for move_base and teleop to use `odom_combined`.

### CODE REVIEW

**Comment by knorth55** [comment]
cc. @sktometometo

**Comment by knorth55** [comment]
upstream merged

**Comment by k-okada** [comment]
@sktometometo sorry could you resolve conflicts?

**Comment by sktometometo** [comment]
@k-okada OK, I will do it

**Comment by sktometometo** [comment]
@k-okada conflicts are resolved.

File: jsk_fetch_robot/jsk_fetch_startup/launch/fetch_teleop.xml [hunk 1]
```diff
@@ -21,7 +22,8 @@
        joystick_teleop need to publish /teleop/cmd_vel/unsafe ,
        see https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_fetch_robot/jsk_fetch_startup/README.md -->
   <node name="teleop" pkg="fetch_teleop" type="joystick_teleop" respawn="true">
-    <remap from="teleop/cmd_vel" to="/joy_vel" />
+    <remap from="teleop/cmd_vel" to="/teleop/cmd_vel/unsafe" />
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch_startup/launch/fetch_teleop.xml:25]: sorry but conflict is not properly solve...

Comment (knorth55) [reply jsk_fetch_robot/jsk_fetch_startup/launch/fetch_teleop.xml:25]: I made a PR #1358

File: jsk_fetch_robot/jsk_fetch_startup/launch/fetch_teleop.xml [hunk 2]
```diff
@@ -74,6 +76,50 @@
...
+    inflater:
+      inflation_radius: 1.0 # 0.7
+      cost_scaling_factor: 3.0 # 25.0 default 10, increasing factor decrease the cost value
+    obstacles:
+      min_obstacle_height: 0.05
+    # default 5 (http://wiki.ros.org/navigation/Tutorials/Navigation%20Tuning%20Guide)
+    update_frequency: 10.0
+    footprint_padding: 0.05
+  </rosparam>
+
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch_startup/launch/fetch_teleop.xml:122]: :sob: 
this part, too.

Comment (knorth55) [reply jsk_fetch_robot/jsk_fetch_startup/launch/fetch_teleop.xml:122]: I made a PR #1358

**Pull request merged**